src/backend/aspen/api/views/users.py

```diff
@@ -14,3 +14,15 @@ async def get_current_user(
     request: Request, db: AsyncSession = Depends(get_db), user=Depends(get_auth_user)
 ) -> UserMeResponse:
     return UserMeResponse.from_orm(user)
+
+
+@router.put("/me", response_model=UserMeResponse)
+async def update_user_info(
+    user_update_request: UserUpdateRequest,
+    db: AsyncSession = Depends(get_db),
+    user=Depends(get_auth_user),
+) -> UserMeResponse:
+    user.agreed_to_tos = user_update_request.agreed_to_tos
+    user.acknowledged_policy_version = user_update_request.acknowledged_policy_version
```

Comment:
I see you updated the FE `AcknowledgePolicyChanges` to send both, but I think I prefer the approach of having the various update fields be optional since since pretty soon we're going to allow the user to update other (new) attributes about themself (like that Bio submitting number for GISAID ingest, etc). If you feel strongly that it should be both pieces required and together, maybe make this a sub-route, like `/me/policies`, so that we can update other parts of the user later without any concern for policy agreement stuff.

Reply:
Fair enough. Updated to make all fields optional

Reply:
Great, thanks!